scripts/utils.sh

```diff
@@ -34,6 +40,14 @@ have_compose_v2 () {
   export COMPOSE_CMD
 }
 
+select_compose_file () {
+  if [ ${SCNODE_ROLE:-} = "forger" ]; then
```

Comment:
That will be a syntax error. It has to be(missing double qoutes)

`
if [ "${SCNODE_ROLE:-}" = "forger" ]; then
`